cmd/croc/send.go

```diff
@@ -27,20 +30,27 @@ var code string
 var relayUrl = "https://raw.githubusercontent.com/runpod/runpodctl/main/cmd/croc/relays.json"
 
 var SendCmd = &cobra.Command{
-	Use:   "send [filename(s) or folder]",
-	Args:  cobra.ExactArgs(1),
+	Use:   "send [file0] [file1] ...",
+	Args:  cobra.MinimumNArgs(1),
 	Short: "send file(s), or folder",
 	Long:  "send file(s), or folder to pod or any computer",
-	Run: func(cmd *cobra.Command, args []string) {
-
-		rand.Seed(time.Now().UnixNano())
```

Comment:
no longer necessary (and in fact _reduces_ the randomness rather than enhancing it in `go >= 1.22` : https://go.dev/blog/chacha8rand